src/components/FormComponent/FormComponent.css

```diff
@@ -61,7 +59,6 @@
   color: #fff;
 }
 
-.form-style input[type="submit"]:hover,
-.form-style input[type="button"]:hover{
+.form-submit-style:hover {
```

Comment:
Take a look at how other people name classes, here the "style" part is redundant. I get that it was done in a hurry since it's the end of the workday, but it's better to take a few minutes to think of a good name. Now everything looks good. Proceed on the next day tomorrow.